vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/subdivision_targeting_frequency_cap.cc

```diff
@@ -57,6 +57,10 @@ bool SubdivisionTargetingFrequencyCap::DoesRespectCap(
   const std::string subdivision_targeting_code =
       ads_->get_subdivision_targeting()->GetAdsSubdivisionTargetingCode();
 
+  if (subdivision_targeting_code.empty()) {
+    return false;
```

Comment:
In general, should `DoesRespectCap` return false if the data is malformed? My impression is that we should return true.

Reply:
If subdivision targeting has not been initialized we need to return false so that the ad is not shown outside of the intended region. Thanks